Tasks/Kaplunou/Task5/Task5.py

```diff
@@ -0,0 +1,28 @@
+res=0
+
+def list_sum(l):      
+    for i in l:
+        if type(i)!=list:
+            global res
```

Comment:
why do you need this abomination??? do you not have a way to work without storing this value in a global variable??? -10 points for the logical mistake